### PR TITLE
[jstl-api] Initial Integration for jstl-api

### PR DESCRIPTION
**Pull request by aschaich**

### CODE REVIEW

File: projects/jstl-api/Dockerfile [hunk 1]
```diff
@@ -0,0 +1,59 @@
...
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+#
+################################################################################
+
+FROM gcr.io/oss-fuzz-base/base-builder-jvm
+
+#
+# install maven configuration, which is also used by gradles's publishToMavenLocal
```
Comment (jonathanmetzman) [inline projects/jstl-api/Dockerfile:20]: begin with captial, end with period.

File: projects/jstl-api/Dockerfile [hunk 2]
```diff
@@ -0,0 +1,59 @@
...
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+#
+################################################################################
+
+FROM gcr.io/oss-fuzz-base/base-builder-jvm
+
+#
```
Comment (jonathanmetzman) [inline projects/jstl-api/Dockerfile:19]: remove this line

File: projects/jstl-api/Dockerfile [hunk 3]
```diff
@@ -0,0 +1,59 @@
...
+# install maven configuration, which is also used by gradles's publishToMavenLocal
+#
```
Comment (jonathanmetzman) [inline projects/jstl-api/Dockerfile:21]: remove this line

File: projects/jstl-api/Dockerfile [hunk 4]
```diff
@@ -0,0 +1,59 @@
...
+# clone repository
+#
+RUN git clone https://github.com/eclipse-ee4j/jstl-api.git ${LIBRARY_NAME}
+
+#
+# apply fixes
+#
+#ADD *.patch ${SRC}/
+#RUN cd ${SRC}/${LIBRARY_NAME} && (for i in ${SRC}/*.patch; do tr -d '\015' < $i | git apply -v; done )
+
```
Comment (jonathanmetzman) [inline projects/jstl-api/Dockerfile:56]: Is this needed?

File: projects/jstl-api/build.sh [hunk 5]
```diff
@@ -0,0 +1,92 @@
+#!/bin/bash -eu
+# Copyright 2022 Google LLC
```
Comment (jonathanmetzman) [inline projects/jstl-api/build.sh:2]: 2023

**Review by jonathanmetzman — APPROVED** [review]
lgtm

**Comment by aschaich** [comment]
I opened a new pull request (#9570) to address the commented issues

**Pull request merged**